recipes/r-cmdstanr/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "r-cmdstanr" %}
+{% set version = "0.6.1" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source: 
+  url: https://github.com/stan-dev/cmdstanr/archive/v{{ version }}.tar.gz
+  sha256: 493972deb6a915918e2f24a55b7c9f67c7c140bcd103239716318fb1283a5ee2
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-r6 >=2.4.0
+    - r-checkmate
+    - r-data.table
+    - r-jsonlite >=1.2.0
+    - r-posterior >=1.1.0
+    - r-processx >=3.5.0
+    - r-withr >=2.5.0
+
+  run:
+    - r-base
+    - r-r6 >=2.4.0
+    - r-checkmate
+    - r-data.table
+    - r-jsonlite >=1.2.0
+    - r-posterior >=1.1.0
+    - r-processx >=3.5.0
+    - r-withr >=2.5.0
+
```

Since the intent of this package is solely to interface with CmdStan, I would argue that this we should guarantee its installation, rather than having the users sideload it with the `install_cmdstan()` function. The `cmdstan` package appears to already set the `CMDSTAN` environment variable, so this should automatically pick it up.

```suggestion
    - cmdstan

```

---

Concur, this makes the most sense.